<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLStu</class>
 <widget class="QWidget" name="SQLStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLStu</string>
  </property>
  <widget class="QPushButton" name="pushButton_insert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_deleteid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>id:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_insertname">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_newid">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1001</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_updateid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>id:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_insertid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_insertage">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>age:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newname">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_deleteid">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1001</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_insertname">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_insertid">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1001</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_newage">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>age</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>156</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_newname">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_insertage">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tablename">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_newage">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>521</width>
     <height>211</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
  </widget>
  <widget class="QSpinBox" name="spinBox_updateid">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1001</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_tablename">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>表名:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_newid">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>290</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
